soundcrate/src/app/user/[username]/list/[list_id]/page.js

```diff
@@ -12,6 +14,15 @@ export default function ListPage({ params }) {
   const list_data = get_list_by_id(list_id);
   const song_ids = get_list_song_ids(list_id);
 
+  const router = useRouter();
+  const session = useSession();
+  useEffect(() => {
+    if (session?.status == "unauthenticated"){
+      router.replace("/");
+    }
+  }, [session, router]);
+
+
```

Comment:
I think we planned for unauthenticated users to be able to view the user pages - the only ones they shouldn't be able to see would be any screens related to updating user data and the modals for writing reviews/creating lists.

Reply:
okay i'll change that